src/main/java/org/openmrs/maven/plugins/model/Server.java

```diff
@@ -0,0 +1,132 @@
+package org.openmrs.maven.plugins.model;
+
+/**
+ * Class for Server model
+ */
+public class Server {
+    private String serverId;
+    private String dbPath;
+    private String dbDriver;
+    private String dbUri;
+    private String dbUser;
+    private String dbPassword;
+    private String interactiveMode;
+    private String version;
+
+    public static class ServerBuilder {
+        private String nestedServerId;
+        private String nestedDbDriver;
+        private String nestedDbUri;
+        private String nestedDbUser;
+        private String nestedDbPassword;
+        private String nestedInteractiveMode;
+        private String nestedVersion;
+
+        public ServerBuilder setNestedInteractiveMode(String nestedInteractiveMode) {
```

Comment:
"nested" is confusing. I would not use it. Typically you make builder setters match class' getters.